### PR TITLE
create email excludes instant-subscribed topics from digest

### PR DESCRIPTION
**Pull request by laurieboyes**
also wrote some tests for fetch-user-follows

### CODE REVIEW

File: functions/create-email/lib/fetch-user-follows.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ const metrics = require('../lib-shared/metrics');
 const tap = require('../lib-shared/utils/tap');
 
 module.exports = function fetchUserFollows (userId) {
-	logger.info('Fetching user follows', { event: 'FETCH_USER_FOLLOWS', user: userId });
+	logger.info('Fetching user follows', { event: 'FETCH_USER_FOLLOWS', user: userId, myFtApiUrl: process.env.MYFT_API_URL });
 	return myFtClient
 			.getAllRelationship('user', userId, 'followed', 'concept')
 			.then(tap(() => metrics.count('fetchUserFollows')))
-			.then(res => res.items)
+			.then(res => res.items.filter(item => !item._rel.instant))
```
Comment (laurieboyes) [inline functions/create-email/lib/fetch-user-follows.js]: Not sure how safe this is if we're definitely considering storing the instant property as a `"true"` string. My worries are that 6 months down the line we'll think we're safe to turn them off by setting them to `"false"` (although, written down, that does sound a bit paranoid). 

My preference is that we do store the instant prop as a boolean, but I've not quite grokked why this is awkward to achieve.

Comment (keirog) [inline functions/create-email/lib/fetch-user-follows.js]: I feel slightly iffy about this too, but no strong objection

**Comment by keirog** [comment]
👍

File: functions/create-email/lib/fetch-user-follows.js (hunk 1 above)
Comment (laurieboyes) [inline functions/create-email/lib/fetch-user-follows.js]: Good enough for me! Cheers 👍

**Pull request merged**